src/Rule/Boolean.php

```diff
@@ -39,9 +39,6 @@ class Boolean extends Rule
      */
     public function validate($value)
     {
-        if ($value === true || $value === false) {
-            return true;
-        }
-        return $this->error(self::NOT_BOOL);
+        return is_bool($value) ?: $this->error(self::NOT_BOOL);
```

Comment:
Its classic ternary operator case. For me personaly not using short ternary operator there same thing like avoid use built-in functions or language features.